decidim/decidim
# Fix Exception as admin on a Proposal with meeting author

#### :tophat: What? Why?
When an  administrator, visits a Proposal of which author is a Meeting, I see an exception (error 500)

#### :pushpin: Related Issues
*Link your PR to an issue*
- Fixes #10623

#### Testing
1. Sign in as admin
2. Go to a Proposal whose author is a Meeting
3. See exception
4. Apply patch
5. Go to a Proposal whose author is a Meeting, see there is no error. 

:hearts: Thank you!
